docs/integration-guides/key-management.md

```diff
@@ -454,17 +454,24 @@ curl -d '{
 }
 ```
 
-!!! info
-    Below are a few helpful pieces of information to consider:
+!!! tip "Block watching and re-work"
+    Since V20.0, blocks processed using [`process`](/commands/rpc-protocol/#process) are placed under observation by the node. If you wish to disable this feature, add `"watch_work": "false"` to the process RPC command.
+
+    * If a block is not confirmed within a certain amount of time (configuration option `work_watcher_period`, default 5 seconds), an **automatic re-generation of the block's proof-of-work** may take place.
+    * Re-generation only taskes place when the network is saturated, and the proof-of-work is used to prioritize blocks.
```

Comment:
"Re-generation only takes place when the network is unable to confirm transactions quickly enough (commonly referred to as the network being "saturated"), and the higher proof-of-work is used to help prioritize the block higher in the processing queue."